build.wake

```diff
@@ -154,11 +154,40 @@ def tarball Unit =
     # Identify those sources files to include in the tarball
     require Pass allSources = sources "." `.*`
 
-    def srcs =
+    def skipFolders =
+        (".circleci/", "debian/", ".github/", ".vscode/", Nil)
+        | catWith "|"
+
+    def skipFiles =
+        (
+            ".clang-format",
+            ".dockerignore",
+            ".git-blame-ignore-revs",
+            ".gitattributes",
+            "wake.spec.in",
+            ".wakemanifest",
+            Nil
+        )
+        | catWith "|"
+
+    def skipFilesRecursive =
+        (".gitignore", Nil)
+        | catWith "|"
+
+    require Pass regex =
+        "(({skipFolders}).*|({skipFiles})|(.*/)*({skipFilesRecursive}))"
+        | stringToRegExp
+
+    def Pair skips srcs =
         allSources
-        | filter (! matches `(debian/|wake.spec).*` _.getPathName) # omit packaging files
-        | filter (! matches `(\.circleci/|\.github/).*|\.dockerignore` _.getPathName) # omit CI files
-        | filter (! matches `(.*/)*\.gitignore` _.getPathName) # omit git files
+        | splitBy (matches regex _.getPathName)
+
+    def _ =
+        def files = (skips | map (\x (("  ", x.getPathName, Nil) | cat)))
+        def msg = "Files below are excluded from tarball", files
+
+        msg
+        | map (println _)
```

Comment:
Do you want to leave this in or was it a debugging print?

Reply:
I wanted to leave it in as an info since `make tarball` is pretty much only called in CI  where its okay to be a bit noiser. Could also dump it to logEcho if preferred

Reply:
I'd prefer `info`; it's distinctly *not* the command line itself as is intended for `echo`.  However, it's currently being sent to `report` -- which isn't a bad place either, but it might be noisier than you intended.

Reply:
the specific function is very rarely called by users, instead its only called by CI so we can be more tolerance to noise here but I don't feel super strongly and I see that I'm in the minority so I'll move it over to `info`

Reply:
I don't actually mind it being a `report`, I just want it to be either that or `info` rather than `echo`. Sorry for the confusion!